tests/resources/tracing/nodestats/catenanes_nodestats_grainstats.csv

```diff
@@ -1,3 +1,3 @@
 ,image,grain_number,num_crossings,avg_crossing_confidence,min_crossing_confidence
-grain_0,test_image,0,4,0.4013589828832889,0.2129989376767838
-grain_1,test_image,1,4,0.3441057054647598,0.17063184531586506
+grain_0,test_image,0,4,0.21426594097881008,0.001258249874731443
```

Comment:
Are we happy with these changes? They seem quite large.

Reply:
I was suspicious of these changes too, however I looked at the traces and the traces are fine, though they slightly different (due to randomness Max and I believe). the differences in the traces are not significant, but they do result in some quite different results as you can see. The crossings in question for that image are very difficult and a slight change in trace does produce a large change in results, however upon manual review, we believe these changes do reflect the slightly altered (but not problematic trace)

so TLDR: yes, caused by a small change in the trace due to randomness, turns out the tracing for that image is actually slightly better now (but that's just luck)